### PR TITLE
feat: Support bundle v1beta1 updates

### PR DESCRIPTION
**Pull request by c-ryan-k**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

File: azext_edge/edge/providers/support/base.py [hunk 1]
```diff
@@ -406,7 +419,8 @@ def process_cluster_role_bindings(
         label_selector=label_selector, field_selector=field_selector
     )
     for binding in cluster_role_bindings.items:
-        namespace = getattr(binding.metadata, "annotations", {}).get("meta.helm.sh/release-namespace")
+        annotations = getattr(binding.metadata, "annotations", {}) or {}
```
Comment (digimaun) [inline azext_edge/edge/providers/support/base.py:420]: This pattern because "annotations" could be None?

**Review by digimaun — APPROVED** [review]

File: azext_edge/edge/providers/support/base.py (hunk 1 above)
Comment (c-ryan-k) [reply azext_edge/edge/providers/support/base.py:420]: Yup
image

I ran a few bundles before I noticed some files missing even with an `or 'test'` at the end of the get, turns out some of them are just `None`

File: azext_edge/edge/providers/support/base.py [hunk 2]
```diff
@@ -533,11 +547,98 @@ def process_cron_jobs(
     )
 
 
+def _get_broker_namespace() -> str:
```
Comment (digimaun) [inline azext_edge/edge/providers/support/base.py]: There is a function `get_mq_namespaces` in this module that had similar goals but its simpler with less safety and no mem cache.
